src/main/scala/fs2/async/package.scala

```diff
@@ -0,0 +1,39 @@
+package fs2
+
+import fs2.util.Catchable
+
+
+/**
+ * Created by pach on 10/10/15.
+ */
+package object async {
+
+
+  /**
+   * Create a new continuous signal which may be controlled asynchronously.
+   * Note that this would block any resulting processes (discrete, continuous) until any signal value is set.
+   */
+  def signalUnset[F[_]:AsyncExt,A]: Stream[F,mutable.Signal[F,A]] =
```

Comment:
I'm not sold on having pipe on Signal. It is weird to have a Signal whose continuous portion isn't actually continuous. It also raises lots of questions... What is the value of a signal when the output is dropped? The previous output? Block until the next output?

I no longer like idea of unset signals. If that concept is useful, it seems like it belongs as a separate type I think.

If you just need to pass around a pair of streams that are formed from some signals, that is fine, but I like keeping signal to have a clear meaning.